src/kem/kem.c

```diff
@@ -810,7 +836,7 @@ OQS_API OQS_KEM *OQS_KEM_new(const char *method_name) {
 		return NULL;
 	}
 }
-
+#include <stdio.h>
```

Comment:
Seems like a spurious `#include`; not a big deal but if you're doing another commit please remove it.

Reply:
removed